aws/iam/ses_send/main.tf

```diff
@@ -1,31 +1,24 @@
-resource "aws_iam_group" "mod" {
```

Comment:
So, do you intend on adding the group resource to the users of this module? Do we need to migrate that?

Reply:
With this, the users don't need the group. They now get directly attached to the policy through the attachment.

There was already a `users` variable, so I believe nothing needs to change on the terraform declaration other than the ref to upgrade, but it'll make modifications inside AWS.

Does that answer your question?

Reply:
Yeah, I think so. I do think we should find the usages and apply them once you make this change though, because it will involve us deleting an actual group in AWS...which is a good thing